exec/java-exec/src/main/resources/rest/query/query.ftl

```diff
@@ -111,6 +111,12 @@
       enableBasicAutocompletion: true,
       enableLiveAutocompletion: false
     });
+    // meta-enter to submit query
+    document.getElementById('queryForm')
+            .addEventListener('keydown', function(e) {
+      if (!(e.keyCode == 13 && e.metaKey)) return;
```

Comment:
For Windows machines, the `metaKey` equivalent is the Windows key. Interestingly, this is not supported by Firefox 48+ (Ref: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey )
For Windows, the key combination is `Ctrl+Enter` .  (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/ctrlKey)
So, you'll need to do a check for the key events something like this:
```
if (! ((e.ctrlKey || e.metaKey) &&  e.keyCode == 13) ) return;
```

Reply:
Folks familiar with Mac tell me the Meta key has been deprecated and since Ctrl is available on Mac, let's go with that combination: `Ctrl + Enter`
Also, check for `keyCode == 10` (Refer: https://bugs.chromium.org/p/chromium/issues/detail?id=79407 )
This doesn't seem to be listed anymore, but might be worth having to ensure compatibility.